README.md

```diff
@@ -88,29 +88,30 @@ Here's the generated DOM tree, as viewed in Chrome inspector:
 A couple of things to note from the code:
 
 1. There are no framework classes to inherit and extend from.
-2. There are no API method calls.
-3. There are no HTML body tags.
-4. All we have is a single JSON-like variable.
-5. The DOM just builds itself without you running any function.
+2. There are no API method calls. The only API is a set of six `$` keywords.
```

Comment:
This is confusing because it doesn't define what a '$' keyword is. I would suggest something like "There are no API method calls. You just construct vanilla javascript objects with specially named ('$' prefix) properties which are detected by Cell."